ui/component/commentMenuList/view.jsx

```diff
@@ -9,12 +9,17 @@ import * as ICONS from 'constants/icons';
 import * as MODALS from 'constants/modal_types';
 import ChannelThumbnail from 'component/channelThumbnail';
 import Icon from 'component/common/icon';
+import classnames from 'classnames';
 import React from 'react';
 import { useIsMobile } from 'effects/use-screensize';
+import { NavLink } from 'react-router-dom';
+import { formatLbryUrlForWeb } from 'util/url';
 
 type Props = {
   uri: ?string,
   authorUri: string, // full LBRY Channel URI: lbry://@channel#123...
+  authorName: string,
```

Comment:
`authorName`

There's one instance where this isn't provided. Is it missing, or optional (i.e. change to `authorName?: string`

```
                  <CommentMenuList
                    uri={uri}
                    isTopLevel={isTopLevel}
                    isPinned={isPinned}
                    commentId={commentId}
                    authorUri={authorUri}
                    commentIsMine={commentIsMine}
                    handleEditComment={handleEditComment}
                    supportAmount={supportAmount}
                    setQuickReply={setQuickReply}
                  />
```

Reply:
It's not provided.